package.json

```diff
@@ -37,67 +41,68 @@
   },
   "license": "MIT",
   "funding": "https://github.com/sponsors/philsturgeon",
-  "main": "lib/index.js",
-  "typings": "lib/index.d.ts",
-  "exports": {
-    "types": "./lib/index.d.ts",
-    "require": "./cjs/index.js",
-    "default": "./lib/index.js"
-  },
+  "types": "dist/lib/index.d.ts",
+  "main": "dist/lib/index.js",
   "browser": {
     "fs": false
   },
   "engines": {
-    "node": ">= 17"
+    "node": ">= 16"
   },
   "files": [
     "lib",
+    "dist",
     "cjs"
   ],
-  "type": "module",
   "scripts": {
-    "build": "swc ./lib/*.js -d ./cjs && swc ./lib/**/*.js -d ./cjs && copyfiles ./lib/**/*.cjs -u 1 ./cjs && node ./scripts/writeCJSPackageJSON.cjs",
-    "clean": "shx rm -rf coverage cjs",
-    "lint": "eslint lib test/fixtures test/specs",
-    "test": "npm run test:node && npm run test:typescript && npm run test:browser && npm run lint",
-    "test:node": "mocha",
-    "test:browser": "cross-env NODE_OPTIONS=--openssl-legacy-provider karma start karma.conf.cjs --single-run",
-    "test:typescript": "tsc --noEmit --strict --lib esnext,dom test/specs/typescript-definition.spec.ts",
-    "coverage": "npm run coverage:node && npm run coverage:browser",
-    "coverage:node": "c8 node_modules/mocha/bin/mocha",
-    "coverage:browser": "npm run test:browser -- --coverage",
+    "prepublishOnly": "yarn build",
+    "lint": "eslint src",
+    "build": "rm -fr dist/* && tsc",
+    "typecheck": "tsc --noEmit",
+    "prettier": "prettier --write \"**/*.+(js|jsx|ts|tsx|har||json|css|md)\"",
+    "test": "vitest --coverage",
+    "test:node": "yarn test",
+    "test:browser": "cross-env BROWSER=\"true\" yarn test",
+    "test:update": "vitest -u",
+    "test:watch": "vitest -w",
     "upgrade": "npm-check -u && npm audit fix"
   },
   "devDependencies": {
-    "@babel/polyfill": "^7.12.1",
-    "@chiragrupani/karma-chromium-edge-launcher": "^2.2.2",
-    "@jsdevtools/eslint-config": "^1.0.7",
     "@jsdevtools/host-environment": "^2.1.2",
-    "@jsdevtools/karma-config": "^3.1.7",
```

Comment:
Thank you, this is helping me deprecate those older @jsdevtools packages that were mainly used for APIDevTools packages. https://www.npmjs.com/package/@jsdevtools/karma-config

Reply:
I can remove the host-env one too, since the environment is always running in node.